packages/hop-node/src/gasboost/GasBoostTransaction.ts

```diff
@@ -355,74 +470,60 @@ class GasBoostTransaction extends EventEmitter implements providers.TransactionR
   }
 
   private async boost (item: InflightItem) {
-    const gasPrice = await this.getBumpedGasPrice()
-    const maxGasPrice = parseUnits(this.maxGasPriceGwei.toString(), 9)
-    if (gasPrice.gt(maxGasPrice)) {
+    const gasFeeData = await this.getBumpedGasFeeData()
+    const maxGasPrice = this.parseGwei(this.maxGasPriceGwei)
+    const priorityFeePerGasCap = this.parseGwei(this.priorityFeePerGasCap)
```

Comment:
It looks like `this.priorityFeePerGasCap` is not set in the constructor or variable instantiation. If it is `undefined`, I believe `this.parseGwei()` would fail. On https://playground.ethers.org/

```
parseUnits(undefined, 9)
"value must be a string (argument=\"value\", value=undefined, code=INVALID_ARGUMENT, version=units/5.4.0)"
```

Is it expected that this is always set with the `setter`? If not, maybe we set this in the constructor

Reply:
it's definitely being set in the constructor

![DeepinScreenshot_select-area_20210914141526](https://user-images.githubusercontent.com/168240/133335019-2322c7f4-938f-4bfa-a0bb-3ef426fedf80.png)

constant 

![DeepinScreenshot_select-area_20210914141542](https://user-images.githubusercontent.com/168240/133335023-1230f8f4-50bb-4cc7-b2f3-c5a033125737.png)

Reply:
Oversight on my part. You are correct -- thx for pointing that out. Will resolve this now.